tests/cyclers/integer_exclusions_advanced/suite.rc

```diff
@@ -0,0 +1,31 @@
+[cylc]
+    [[reference test]]
+        live mode suite timeout = PT3M
+[scheduling]
+    initial cycle point = 1
+    final cycle point = +P15  # = 16
+    runahead limit = P12
+    cycling mode = integer
+    [[dependencies]]
+        [[[ R/P1!(2,3,7) ]]]
+            graph = foo
+        [[[ P1 ! P2 ]]]
+            graph = woo
+        [[[ P1 ! +P1/P2 ]]]
+            graph = bar
+        [[[ P1 !(P2,6,8) ]]]
+            graph = qux
+[runtime]
```

Comment:
Could we add a test for implicit start/end context for integer cycling as well e.g. `R/1/P2!P3`.